tests/testthat/test-utils.R

```diff
@@ -45,33 +45,217 @@ test_that("dims to col & row and back", {
   expect_equal(exp, got)
 
 })
+test_that("`wb_dims()` works/errors as expected with unnamed arguments", {
 
-test_that("wb_dims() works", {
+  # Acceptable inputs
+  expect_equal(wb_dims(), "A1")
+  expect_equal(wb_dims(NULL), "A1") # to help programming with `wb_dims()` maybe?
```

Comment:
This is a bit strange, but ok

Reply:
Would you prefer something else? I just wanted to have edge cases to work anyway..

Reply:
I'd assume that it returns NULL or even stop, because otherwise the wb function will complain. IMHO this could let the user know that something is wrong on their side